gbtis/Kinect.cs

```diff
@@ -93,6 +101,18 @@ public class Kinect {
         }
 
         /// <summary>
+        /// Gets the Head Position of the Active Body
+        /// </summary>
+        /// <returns>The head position of the active body, or null if 
+        /// no body is currently being tracked
+        /// </returns>
+        public CameraSpacePoint? ActiveBodyHeadPosition() {

```

Comment:
It's for the halo thing I had it wrote before we decided we werent doing it anymore